### DISCUSSION TITLE
Exclude library "livetv" ?

### DISCUSSION THREAD

**Discussion by theChad6** [Taxel/PlexTraktSync#768] [Q&A] [answered]
PlexTraktSync [0.18.15]
Plex Version 1.24.4.5081 (Windows 10)
Synology NAS : DSM7 : Docker

Is there any way to ignore Live TV?  While playing LiveTV, these errors get pumped out every 10 seconds.
Adding this snipit to config.json doesn't seem to work.
` "excluded-libraries": [
        "Private",
        "Family Holidays",
                "Home Movies",
                "livetv"
    ]`

theChad@synnas:/volume1/docker/appdata$ sudo docker-compose run --rm plextraktsync
Creating trash-guides_plextraktsync_run ... done
Listening for events!
INFO: Starting AlertListener: wss://192-168-0-16.af24ef612f1342c19cb45edd07a8fd14.plex.direct:32400/:/websockets/notifications?X-Plex-Token=<hidden>
ERROR: <Episode:/livetv/sessions/a1870327-dd48-491c-a54c-478a986e0325:FOX-10-Arizona-Morni-s2022eNone>: Unable to parse a valid provider from guid tv.plex.xmltv://episode/FOX%2010%20Arizona%20Morning/1900-01-01%2000%3A00%3A00
ERROR: <Episode:/livetv/sessions/a1870327-dd48-491c-a54c-478a986e0325:FOX-10-Arizona-Morni-s2022eNone>: Unable to parse a valid provider from guid tv.plex.xmltv://episode/FOX%2010%20Arizona%20Morning/1900-01-01%2000%3A00%3A00

**Comment by glensc** [comment]
Feature requests need to post as issues.

**Comment by glensc** [comment]
Bug reports need to be posted as issues.

**Comment by glensc** [comment]
One question mark is enough!

**Reply by theChad6** [reply]
Apologies.  Issue/feature created. Extra "?" removed from title.

**Reply by glensc** [reply]
you forgot link to issue so people reaching here could follow there

**Comment by theChad6** [comment, marked as answer]
Converted to [Issue #770](https://github.com/Taxel/PlexTraktSync/issues/770#issue-1121062635)
